docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,169 @@
+
+If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.
+
+Finally, it's time to add the details of your highlight. You can add a description yourself or use the auto-summarize function to generate a description.
+
+![gif of creating a highlight](../../static/gif/highlight.gif)
+
+### Writing an Effective Highlight
+
+Writing an effective highlight is key to attracting contributors to your project. Here are some tips to help you write an effective highlight:
+
+#### Highlighting Issues
+
+1. **Focus on the problem solved**: Begin by briefly describing the problem or challenge the issue addresses. This sets the context for why it was important.
+2. **Acknowledge contributor(s)**: Mention the contributor(s) who reported or worked on the issue, recognizing their effort.
+3. **Impact statement**: Explain how resolving this issue benefits the project, such as improving functionality, user experience, or security.
+
+#### Highlighting PRs
+
+1. **Summarize the changes**: Start with a clear summary of what the PR achieves. This could be a new feature, bug fix, or enhancement.
+2. **Credit the contributors**: Acknowledge everyone who contributed to the PR, including reviewers.
+3. **Highlight the value**: Explain the value of these changes to the project or its users. For instance, how it improves performance, adds a requested feature, or simplifies the codebase.
+
+#### Highlighting Dev.to Blog Posts
+
+1. **Capture the essence**: Begin with an interesting blog post summary, capturing the main theme or message.
+2. **Discuss the relevance**: Explain why this blog post is important for your open source project or community. Is it about a major update, a community event, or sharing best practices?
+3. **Engage with a call-to-action**: Encourage readers to engage with the post, whether by reading, commenting, or sharing their thoughts on the topic.
+
+## Insights: Connecting Your Repositories
+
+OpenSauced Insights you to see the impact of your contributions. You can use insights pages to track repositories' growth, analyze work, and connect with others.
+
+## Why Create an Insights Page
+
+- Personal Contribution Tracking
+- Skill Development and Portfolio Building: Contributing to open source projects can be a great way to develop new skills. Your contributions can also be used as samples for your portfolio. Looking for ways to implement your open source experience in your job search journey? Check out our [job search guide](https://docs.opensauced.pizza/job-seekers-guide/job-seekers-guide-introduction/)! 😄
+- Identifying Potential Projects to Contribute To: Finding open source projects to contribute to can be difficult and this feature provides you access to a whole plethora of projects for you to chose from.
+Networking and Community Engagement - explanation
+- Curating Collections for Interest Groups (e.g., top open source React projects)
```

Comment:
I think you might want to revisit this list. The "Networking and community engagement" has no explanation yet. And let's bold them.

@BekahHW I reviewed some here and need your thoughts if this is on the right track. Thanks!

```suggestion
- **Tracking personal contribution**: Add repositories that you contributed to and track your contributions.
- **Skill development and portfolio building**: Contributing to open source projects can be a great way to develop new skills. Your contributions can also be used as samples for your portfolio. Looking for ways to implement your open source experience in your job search journey? Check out our [job search guide](../job-seekers-guide/introduction.md)!
- **Identifying potential projects to contribute to**: Finding open source projects can be difficult. The Insights feature can help you identify projects you would likely contribute to.
- **Networking and community engagement** - explanation
- **Curating collections for interest groups**: Curate repositories that interest you, e.g., top open source React projects.
```

Reply:
Definitely on the right track. This information could be useful both here and other places in the guide: https://dev.to/opensauced/choose-your-own-adventure-in-open-source-paths-to-success-167p